README.md

```diff
@@ -27,91 +19,24 @@ There are several scripts and helpers in this repository a release
-```
-$ release-notify <release version>
-```
-
-### Release with --nomock
-
-The above example workflows run *mock* versions of the release.  To produce
-a fully deployed and announced release, add `--nomock` to the command line.
-
-
-## All Tools
-
-* [gcbmgr](https://github.com/kubernetes/release/blob/master/gcbmgr) : GCB manager - find status and initiate builds
-* [anago](https://github.com/kubernetes/release/blob/master/anago) : Release Tool
-* [branchff](https://github.com/kubernetes/release/blob/master/branchff) : Fast-forward branching helper
-* [find_green_build](https://github.com/kubernetes/release/blob/master/find_green_build) : Ask Jenkins for a good build to use
-* [release-notes](https://github.com/kubernetes/release/blob/master/cmd/release-notes) : Scrape github for release notes \(See below for more info\)
-* [prin](https://github.com/kubernetes/release/blob/master/prin) : What tags/releases is my PR IN?
-* [changelog-update](https://github.com/kubernetes/release/blob/master/changelog-update) : Update CHANGELOG.md version entries by rescanning github for text and label changes
-* [push-build.sh](https://github.com/kubernetes/release/blob/master/push-build.sh) : Push a developer (or CI) build up to GCS
-* [script-template](https://github.com/kubernetes/release/blob/master/script-template) : Generate a script template in the kubernetes/release ecosystem
+Tools | Description
+ :---: | --
+[`gcbmgr`](https://github.com/kubernetes/release/blob/master/gcbmgr) | Google Cloud Builder Manager: <br/><br/> This is the main entry point for release managers for producing releases. All release types can be staged and later released using this method.
+[`anago`](https://github.com/kubernetes/release/blob/master/anago) | Release Tool: <br/><br/> The main driver for creating staged builds and releases. This is what runs inside GCB after a job is submitted using `gcbmgr`.
+[`branchff`](https://github.com/kubernetes/release/blob/master/branchff) | Fast-forward branching helper : <br/><br/> A tool used to pull new patches onto the release branch.
+<br/> [`find_green_build`](https://github.com/kubernetes/release/blob/master/find_green_build) <br/><br/> | Asks Jenkins for a good build to use.
+<br/> [`release-notes`](https://github.com/kubernetes/release/blob/master/cmd/release-notes) <br/><br/> | Scrape GitHub for release notes. See [Release Notes Gathering](#release-notes-gathering) for more information.
+<br/> [`prin`](https://github.com/kubernetes/release/blob/master/prin) <br/><br/> | What tags/releases is my PR IN?
+<br/> [`changelog-update`](https://github.com/kubernetes/release/blob/master/changelog-update) <br/><br/> | Updates [CHANGELOG.md](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md) version entries by rescanning github for text and label changes.
+<br/> [`push-build.sh`](https://github.com/kubernetes/release/blob/master/push-build.sh) <br/><br/> | Pushes a developer (or CI) build up to GCS.
```

Comment:
This one was also verbosely copied over, does CI mean the CI-Signal team?

Reply:
"CI" in this context probably means "the CI system". In [the description of the script][desc] they e.g. talk about jenkins, which as far as I know was the precursor of [prow]. I have never used that script and consider it outdated.

[desc]: https://github.com/kubernetes/release/blob/9fe1ffac23fd77cc0459ff5d156fa2c526125587/push-build.sh#L32-L41
[prow]: https://git.k8s.io/test-infra/prow

Reply:
Oh okay, I get it now. I completely misinterpreted what that sentence meant,  looking at `push-build.sh` when the sentence says, "developer" it actually means developer build and CI for Jenkins continuous builds ( exactly what you've pointed out :sweat_smile: ).